mailpoet/lib/PostEditorBlocks/WooCommerceBlocksIntegration.php

```diff
@@ -95,7 +95,7 @@ public function registerCheckoutFrontendBlocks($integration_registry) {
       [
       'defaultText' => $this->settings->get('woocommerce.optin_on_checkout.message', ''),
       'optinEnabled' => $this->settings->get('woocommerce.optin_on_checkout.enabled', false),
-      'defaultStatus' => $this->woocommerceSubscription->isCurrentUserSubscribed(),
+      'defaultStatus' => false,
```

Comment:
Good point, I missed that. I pushed a new commit implementing this change. Could you please check? I might have missed something, as it has been a while since the last time that I touched React code.

Reply:
I missed that. Since it was the last commit and it is a small one, I amended it to remove `defaultStatus` from the typescript interface and force pushed.